### ISSUE TITLE
Delete?

### ISSUE THREAD

**Issue by retokromer** [amiaopensource/AMIA#2]
Should I delete this (empty!) repository?

**Comment by kgrons** [comment]
Yes, I believe I made it [semi] in error trying to create an org policy repo. Please delete!
